Dockerfile

```diff
@@ -76,12 +76,10 @@ RUN apt-get update \
     curl \
     gnupg \
     lsb-release \
-    libffi-dev \
-    libsodium-dev \
-    python3-dev \
-  && SODIUM_INSTALL="system" python3 -m pip install --no-cache-dir pynacl \
-  # switch back to the package manager version once https://github.com/Azure/azure-cli/issues/7368 is resolved
```

Comment:
https://github.com/Azure/azure-cli/issues/7368 was resolved in 2.46.0, and we are upgrading to the latest 2.62.0 because 2.46.0 is not available in the Ubuntu Noble package repository.